core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -825,7 +825,7 @@ protected Jenkins(File root, ServletContext context, PluginManager pluginManager
                 } catch (BindException e) {
                     new AdministrativeError(getClass().getName()+".tcpBind",
                             "Failed to listen to incoming slave connection",
-                            "Failed to listen to incoming slave connection. <a href='configure'>Change the port number</a> to solve the problem.",e);
+                            "Failed to listen to incoming slave connection. <a href='configureSecurity/'>Change the port number</a> to solve the problem.",e);
```

Comment:
(Not sure that _is_ possible, since this code is not running from a web request. Properly speaking, `AdministrativeError/message.jelly` ought to substitute `"${rootURL}"` with the observed context path.